lua/expression3/extensions/math.lua

```diff
@@ -17,13 +17,32 @@ local extension = EXPR_LIB.RegisterExtenstion("math");
 -- TODO: Add math.pi as constant
```

Comment:
I think it's about adding pi as constant to E3 math library, but E3 doesn't supports constants for now as i know